scripts/zones/Beaucedine_Glacier/npcs/Torino-Samarino.lua

```diff
@@ -17,11 +17,12 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-
     local FoiledAGolem = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.CURSES_FOILED_A_GOLEM)
     local tuningOutProgress = player:getCharVar("TuningOut_Progress")
+    local copMission = player:getCurrentMission(COP)
```

Comment:
```
require("scripts/globals/missions")
```

Reply:
thanks for feedback, it worked without it so i forgot to add em ^^;; thanks Wren